src/dataviews/dataview-model-base.js

```diff
@@ -10,8 +10,8 @@ module.exports = Model.extend({
     url: '',
     data: [],
     columns: [],
-    syncData: true,
-    syncBoundingBox: true,
+    sync_on_source_change: true,
```

Comment:
Yes, that sounds reasonable, wasn't sure when I did the change but figured you'd comment here as you just did. :kissing_heart: